src/containers/ChampionList/ChampionList.js

```diff
@@ -1,28 +1,60 @@
 import React, { useEffect, useState } from 'react';
 import ChampionItem from '../../components/ChampionItem/ChampionItem';
 
-import Grid from '@material-ui/core/Grid';
-import Container from '@material-ui/core/Container';
-
-
-const token = "a28vgv0BfswlbW1U6eSd0r727kw_tdNgkm_OneV5PNpddA_d4KQ";
+import ChampionDetails from '../../components/ChampionDetails/ChampionDetails';
+import Modal from '@material-ui/core/Modal';
+import token from "../../token";
+import "../ChampionList/ChampionList.css"
 
 const ChampionList = () => {
 
     const [champions, setChampions] = useState({ champions: [] });
+    const [isLoading, setIsLoading] = useState(false);
+    const [selectedChampionId, setSelectedChampionId] = useState(null);
+    const [open, setOpen] = useState(false);
 
     useEffect(() => {
         const apiUrl = `https://api.pandascore.co/lol/champions?token=${token}`;
+        setIsLoading(true)
         fetch(apiUrl)
             .then((response) => response.json())
             .then((champions) => {
-                //console.log(champions);
                 setChampions({ champions: champions })
+                setIsLoading(false);
             })
     }, [setChampions])
 
+    const championSelectedHandler = (id) => {
+        setOpen(true)
+        setSelectedChampionId(id)
+    }
+
+    const handleClose = () => {
+        setOpen(false);
+    };
+
+    const championsList = champions.champions !== undefined ? champions.champions.map((champion, key) => {
+
+        return <ChampionItem
```

Comment:
Here you should wrap the `ChampionItem` in the `<Grid item ...>`.

Reply:
Changed.
Thank you.